lib/rucio/tests/test_rse.py

```diff
@@ -1479,3 +1480,24 @@ def test_rse_update_unsupported_option(self):
         with pytest.raises(InputValidationError):
             update_rse(rse_id, parameters={'city': 'Not Berlin', 'non_existing_option': 3})
         assert get_rse(rse_id)['city'] == 'Berlin'
+
+
+def test_get_rse_attribute(rse_factory):
+    _, rse_id = rse_factory.make_mock_rse()
+
+    assert get_rse_attribute(rse_id, "test") is None
+
+    add_rse_attribute(rse_id, "test", "test")
+    assert get_rse_attribute(rse_id, "test", use_cache=False) == "test"
+
+    if ENABLE_CACHING:
+        assert get_rse_attribute(rse_id, "test") is None
```

Comment:
Is this assertion correct? You just set the "test" key to the value "test", wouldn't you expect to get the value "test" back as well?

Reply:
The expected behavior is different, the current one asserts that. I changed it to make it obvious that we want to change it in the future.